client/blocks/authentication/social/index.tsx

```diff
@@ -74,7 +81,11 @@ const SocialAuthenticationForm = ( {
 		// are many places in which the social signup form is rendered based only on the presence of the
 		// `signup/social` config flag.
 		! config.isEnabled( 'desktop' ) && (
-			<div className="auth-form__social">
+			<div
+				className={ classNames( 'card', 'auth-form__social', isLogin ? 'is-login' : 'is-signup', {
+					'is-social-first': isSocialFirst,
+				} ) }
```

Comment:
I think having the `card` class name is not great. You're affected by Card component but not using it. Maybe switch to `Card` instead of div and remove the class name? Otherwise, we won't know how this will behave if we remove/change/update the Card component.